### ISSUE TITLE
Improvements to scheduler

### ISSUE THREAD

**Issue by niks255** [d4rken-org/sdmaid-se#79]
I always felt like scheduler lacked a lot of features. Would love to see them implemented. Some ideas:

- "Battery not lower than" setting with the ability to pick the percentage.

-  "Only when charging" checkbox (which should probably disable the previous setting when enabled)

Maybe it even makes sense to add the ability to launch a schedule via the intent (like in swift backup) from automation apps like tasker.

**Comment by Kunzinostrikes** [comment]
I have noticed that scheduled tasks will not run in background like previous SD maid did? So with SD maid SE it will come on screen and run tasks (kicking me off the other app for a moment)

**Comment by d4rken** [comment]
> So with SD maid SE it will come on screen and run tasks (kicking me off the other app for a moment)

The accessibility based operation? In SD Maid 1 this was disabled for scheduled operations. I thought I disabled it in SD Maid 2 too. Because if the phone is locked at that point, it won't even work :thinking:

**Comment by Kunzinostrikes** [comment]
Yes the accessibility based operation comes up on screen at scheduled time.

**Comment by d4rken** [comment]
Will look into it, I'll disable that like in SD Maid 1.
